packages/core/aqua-to-js/src/generate/header.ts

```diff
@@ -35,12 +35,13 @@ export default function generateHeader(
  */
 ${
   outputType === "ts"
-    ? "import type { IFluenceClient as IFluenceClient$$, CallParams as CallParams$$ } from '@fluencelabs/js-client';"
+    ? "import type { IFluenceClient as IFluenceClient$$, ParticleContext as ParticleContext$$ } from '@fluencelabs/js-client';"
     : ""
 }
 
 import {
     v5_callFunction as callFunction$$,
     v5_registerService as registerService$$,
+    FluencePeer as FluencePeer$$
```

Comment:
Some doc on these would be great.

Maybe aqua-api should provide some of these as constants?

Reply:
I will add some explanatory comment near this line, thanks

_Maybe aqua-api should provide some of these as constants?_
Didn't really understand this

Reply:
why import `FluencePeer$$` where is it used? couldn't find

Reply:
Removed